packages/dbos-compiler/cli.ts

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+
+import { Command } from "commander";
+import path from 'node:path';
+import fs from 'node:fs';
+import fsp from 'node:fs/promises';
+import { generateCreate, generateDrop } from "./generator.js";
+import { parseConfigFile } from '@dbos-inc/dbos-sdk/dist/src/dbos-runtime/config.js'
+import * as pg from 'pg'
+import { type CompileResult, compile } from "./compiler.js";
+
+async function emitSqlFiles(outDir: string, result: CompileResult, appVersion?: string | boolean) {
+  await fsp.mkdir(outDir, { recursive: true });
+
+  const createFile = await fsp.open(path.join(outDir, "create.sql"), "w");
+  try {
+    const append = async (sql: string) => { await createFile.write(sql); };
+    await generateCreate(append, result, appVersion);
+  } finally {
+    await createFile.close();
+  }
+
+  const dropFile = await fsp.open(path.join(outDir, "drop.sql"), "w");
+  try {
+    const append = async (sql: string) => { await dropFile.write(sql); };
+    await generateDrop(append, result, appVersion);
+  } finally {
+    await createFile.close();
+  }
+}
+
+async function deployToDatabase(config: pg.ClientConfig, result: CompileResult, appVersion?: string | boolean) {
+  const client = new pg.default.Client(config);
+  try {
+    await client.connect();
+    console.log(`Deploying to database: ${client.host}:${client.port ?? 5432}/${client.database}`);
+    const append = async (sql: string) => { await client.query(sql); };
```

Comment:
To be clear, this is actually executing the SQL, right? Why is it called append?

Reply:
because I wrote the "write to file" version first, which is actually an append. I couldn't think of a better name and technically, we are "appending" the routine to the database...

Reply:
`executeSql`